pages/02.Tutorials/09.dnsaas/docs.en.md

```diff
@@ -13,6 +13,8 @@ taxonomy:
 OpenStack's Designate provides a Domain Name Service as a service (DNSaaS).
 This means that zones and records can be configured within OpenStack via an API and will be queryable via DNS protocol from public nameservers run by SysEleven.
 This tutorial is intended to make you familiar with the main functionality and aware of the presence of more advanced features of the DNS service of the SysEleven Stack.
+We will show examples with the openstack CLI, discovering the functionality in the [GUI](https://cloud.syseleven.de/horizon/project/dnszones/) is left as an exercise to the user.
```

Comment:
IMO it is not really helpful for the user to tell them what we will do. Either we add examples / links to documentation or reading the text does not really help the user learn something :)